mibitracker/request_helpers.py

```diff
@@ -241,9 +241,9 @@ def copy_run(self, old_label, new_label, **kwargs):
         run_data = {
             'instrument': data['instrument']['id'],  # required
             'slides': ','.join(str(d['id']) for d in data['slide_ids']),
-            'aperture': data['aperture']['id'],  # required
             'label': new_label,
-            'magnification': data['magnification'],  # required, FOV size
+            'fov_size': data['fov_size'],
+            'aperture': data['aperture'] and data['aperture']['id'],
```

Comment:
Aperture also moved to the Image model, we kept it on Run for backwards compatibility but please also specify it at the image level so that if it varies by image that gets captured.